### ISSUE TITLE
Link types not fetched correctly

### ISSUE THREAD

**Issue by kwk** [fabric8-ui/fabric8-planner#2576]
The planner doesn't respect the `workitemlinktypes` relationship in the the space response. Instead it seems to construct the link types URL by hand. Here's an example:

https://api.prod-preview.openshift.io/api/spaces/a7587836-f852-4c5f-9237-ef9814233614/workitemlinktypes

**NOTICE:** This issue became apparent with the advent of space templates. This is because link types are no longer part of a space but part of a space template. Hence, the correct URL for the above request should be taken from the `workitemlinktypes` relationship of the space response:

```sh
curl -s https://api.prod-preview.openshift.io/api/spaces/a7587836-f852-4c5f-9237-ef9814233614 \
| jq '.data.relationships.workitemlinktypes'
```

**Comment by kwk** [comment]
This should be fixed by #2574

**Comment by vikram-raj** [comment]
This issue has been fixed by https://github.com/fabric8-ui/fabric8-planner/pull/2574.
